### PR TITLE
Change export to save

### LINKED ISSUE #7517: UI: "Export order" --> "Save order"
For JabRef 5.3--2021-03-11--8bb7668,
in Options -> Preferences, Tab File, there is a section entitled "Export sort order":
JabRef-Pref-SortOrder

Same in Library -> Library properties:
JabRef-Library-SortOrder

**Is your suggestion for improvement related to a problem? Please describe.**
Currently, these items do not affect the sort order when _exporting_ entries (like using File -> Export).
They alter the sort order when _saving_ the library.

**Describe the solution you'd like**
Make clear that these items affect the saving order by changing "Export [...]" into "Save [>...]"

### PR DESCRIPTION
**Pull request by yinpeiqi**
fix for 7517, change Export to Save
Fixes #7517 

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by mlep** [comment]
This is one of the fastest Issue --> PR I have seen!
Looking at your changes, I think the section title "Export sort order" needs to be modified too (and maybe twice: once for the preferences window, and once for the library window).

**Comment by yinpeiqi** [comment]
Okay I changed it too.

**Comment by Siedlerchr** [comment]
You probably need to adjust the localization in the l10n properties file https://devdocs.jabref.org/getting-into-the-code/code-howtos#using-localization-correctly

**Comment by yinpeiqi** [comment]
Only delete the unused part of l10n is enough, am I right?

**Comment by Siedlerchr** [comment]
Actually this PR is not needed, it's a bug in the underlying functionality. Normally the save order should also be applied for the exports.

**Comment by calixtus** [comment]
Please check your github configuration. The checks all stall and without checks it's way harder for us to test changes made to the codebase!

**Comment by koppor** [comment]
Please keep the distinction of "Export" and "Save". Please change to "neutral" wording to enable use of a common FX control:

grafik

(See also https://github.com/JabRef/jabref/issues/7517#issuecomment-809633110)

**Comment by Siedlerchr** [comment]
@yinpeiqi It would be really nice if you could implement the changes as outlined in the screenshots

**Comment by yinpeiqi** [comment]
Is this change ok?

File: src/main/java/org/jabref/gui/commonfxcontrols/SaveOrderConfigPanel.java [hunk 1]
```diff
@@ -71,9 +71,9 @@ private void initialize() {
     }
 
     public void changeExportDescriptionToSave() {
-        exportInOriginalOrder.setText(Localization.lang("Save entries in their original order"));
-        exportInSpecifiedOrder.setText(Localization.lang("Save entries ordered as specified"));
-        exportInTableOrder.setText(Localization.lang("Save in current table sort order"));
+        exportInOriginalOrder.setText(Localization.lang("Keep original order"));
+        exportInSpecifiedOrder.setText(Localization.lang("Use specified order"));
+        exportInTableOrder.setText(Localization.lang("Use current table sort order"));
     }
```
Comment (calixtus) [inline src/main/java/org/jabref/gui/commonfxcontrols/SaveOrderConfigPanel.java:77]: This method and the call to can be removed entirely.

Comment (Siedlerchr) [reply src/main/java/org/jabref/gui/commonfxcontrols/SaveOrderConfigPanel.java:77]: Good catch! Otherwise it's good from my side,tested it locally

**Comment by yinpeiqi** [comment]
Okay, I remove it.

**Comment by calixtus** [comment]
There is still a unused import, breaking a check style rule.

**Comment by yinpeiqi** [comment]
Well, this time I remove it.

**Review by calixtus — APPROVED** [review]
Looks good to me! Thanks.

**Comment by yinpeiqi** [comment]
You are welcome.

**Review by Siedlerchr — APPROVED** [review]

**Comment by Siedlerchr** [comment]
Thanks again for your contribution!

**Pull request merged**